packages/compiler/src/experimental/typekit/kits/type.ts

```diff
@@ -272,5 +274,39 @@ defineKit<TypekitExtension>({
     isUserDefined(type) {
       return getLocationContext(this.program, type).type === "project";
     },
+    inNamespace(type: Type, namespace: Namespace): boolean {
```

Comment:
TODO: allowing type or namespace to be optional would simplify this code a bunch - would it also provide a more flexible API for users? Should we allow that and just return false if either is undefined?